lua/entities/gmod_wire_expression2/core/ranger.lua

```diff
@@ -33,6 +33,13 @@ local function ResetRanger(self)
 	data.rangerfilter_lookup = table.MakeNonIterable{ [self.entity] = true }
 end
 
+local function IsErrorVector(pos)
```

Comment:
Why do some functions in this file use upper camel case?

Reply:
Cuz it doesn't matter